rpc/openapi/openapi.yaml

```diff
@@ -237,6 +237,31 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+
+  /remove_tx:
```

Comment:
Can this please live only in the other PR?

Reply:
oh yeah, I wondered where that went